remediation/fence-agents-remediation/configuration.md

```diff
@@ -10,7 +10,7 @@ nav_order: 2
 # Fence Agents Remediation Configuration
 
 Fence Agents Remediation (FAR) operator watches *FenceAgentsRemediation* (or *far*) Custom Resource ([CR](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)).
-FAR can run alongside another operator (e.g., Node Health Check Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator#readme)) or another mechnaisam that trigger FAR by automatically creating far CRs using *FenceAgentsRemediationTemplate* (or *fartemplate*) CR. FAR can also be run as standalone, when far CR is generated manually using an administrator’s help.
+Those CRs can be created either manually by cluster administrators (and run as standalone), or by other operators (e.g., Node Health Check Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator#readme)) or different mechnaisams that trigger FAR by automatically creating far CRs using *FenceAgentsRemediationTemplate* (or *fartemplate*) CR.
```

Comment:
> (and run as standalone)

- too verbose IMHO
- also wrong, because "CRs can be created and run as standalone" doesn't make sense

> or different mechnaisams

- typo
- what about "or similar tools"?

Reply:
>what about "or similar tools"?

👍🏻 

>too verbose IMHO
also wrong, because "CRs can be created and run as standalone" doesn't make sense

I was thinking of adding this word (`standalone`) before showing it in the subsection title, so the user will understand what is this subsection.